presentation/src/main/java/com/maxrzhe/presentation/model/ContactItemViewModel.kt

```diff
@@ -0,0 +1,30 @@
+package com.maxrzhe.presentation.model
+
+import android.view.View
+import com.example.data_api.model.Contact
+import com.maxrzhe.presentation.R
+
+data class ContactItemViewModel(
+    val contact: Contact,
+    val onItemClick: (item: ContactItemViewModel) -> Unit,
+) : BaseItemViewModel(), View.OnClickListener {
```

Comment:
we can avoid using View.OnClickListener. for listening clicks we should use data binding and property in class with type () -> Unit. for simplification we can use typealias feature of Kotlin language and create once type alias for ()-> Unit and reuse it every time when we need it

Reply:
fixed